wp-content/themes/csisjti/assets/_scss/layout/_header.scss

```diff
@@ -1 +1,200 @@
+    bottom: 0;
+    width: 320px;
+    background-color: $color-white-100;
+    visibility: hidden;
+    transition: clip-path 0.3s linear, visibility 0.3s linear;
+    clip-path: inset(0 0 0 100%);
+
+    @include breakpoint('large') {
+      position: static !important;
+      top: unset;
+      right: unset;
+      left: unset;
+      z-index: 1;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 100%;
+      height: 100%;
+      padding: 0;
+      background-color: none;
+      transform: none;
+      visibility: visible;
+      clip-path: none;
+    }
+
+    &.is-active {
+      visibility: visible;
+      clip-path: inset(0);
+    }
+
+  }
+
+  &__trigger {
+    position: relative;
+    z-index: 2;
+    display: block;
+    width: 40px;
+    height: 40px;
+    background-color: transparent;
+    border: 0;
+    transform: rotate(0deg);
+    cursor: pointer;
+    transition: 0.5s ease-in-out;
+
+    @include breakpoint('large') {
+      display: none;
+    }
+
+    .line {
+      position: absolute;
+      left: 25%;
+      display: block;
+      width: 100%;
+      max-width: 20px;
+      height: 2px;
+      background: $color-black-100;
+      transform: rotate(0deg);
+      opacity: 1;
+      transition: 0.25s ease-in-out;
+
+      /* stylelint-disable-next-line */
+      &:nth-child(1) {
+        top: 35%;
+      }
+
+      /* stylelint-disable-next-line */
+      &:nth-child(2),
+      &:nth-child(3) {
+        top: 50%;
+      }
+
+      /* stylelint-disable-next-line */
+      &:nth-child(4) {
+        top: 65%;
+      }
+    }
+
+    &:hover .line,
+    &:focus .line {
+      background-color: $color-black-100;
+    }
+
+    &.is-active .line {
+      /* stylelint-disable-next-line */
+      &:nth-child(1),
+      &:nth-child(4) {
+        top: 50%;
+        left: 50%;
+        width: 0%;
+      }
+
+      /* stylelint-disable-next-line */
+      &:nth-child(2) {
+        transform: rotate(45deg);
+      }
+
+      /* stylelint-disable-next-line */
+      &:nth-child(3) {
+        transform: rotate(-45deg);
+      }
+    }
+  }
+
+  ul {
+    position: relative;
+    top: 72px;
+    height: 100%;
+
+    @include breakpoint(large) {
+      position: static;
+      top: 0;
+      display: flex;
+    }
+
+    li {
+      position: relative;
+      display: flex;
+      align-items: flex-end;
+      margin: 12px 0;
+      padding: 12px 32px;
+      @extend %font-ui-text-lg-bold;
+      color: $color-black-155;
+
+      @include breakpoint(large) {
+        height: 50%;
+        margin: 0 12px 12px;
+        padding: 0;
+      }
+
+      &.is-active,
+      &:hover {
+        color: $color-black-100;
+        border-left: 4px solid var(--nav-border-color);
+
+        @include breakpoint (large) {
+          border-top: 4px solid var(--nav-border-color);
+          border-left: unset;
+        }
+      }
+
+      &:hover {
+        border-left-color: $color-primary-blue-700 !important;
```

Comment:
You can remove the usage of `!important` & redeclaring the borders if you modify the `--nav-border-color` property instead. Because `&:hover` is more specific, resetting the value of `--nav-border-color` will take precedence.

```scss
&:hover {
  --nav-border-color: $color-primary-blue-700;
}
```